app/file_list/file_list.controller.js

```diff
@@ -40,5 +40,70 @@
       Tag.add_list(uuids, tag);
       this.tag = '';
     };
+
+      $scope.date_regex = '\\d\\d\\d\\d([-\/]\\d\\d?)?([-\/]\\d\\d?)?';
```

Comment:
Good catch, it was indented deeper in the place from which it originated. Fixed.